README.md

```diff
@@ -100,25 +104,6 @@ python frontend/mindsearch_gradio.py
 streamlit run frontend/mindsearch_streamlit.py
 ```
 
-## 🌐 Change Web Search API
-
-To use a different type of web search API, modify the `searcher_type` attribute in the `searcher_cfg` located in `mindsearch/agent/__init__.py`. Currently supported web search APIs include:
-
-- `GoogleSearch`
-- `DuckDuckGoSearch`
-- `BraveSearch`
-- `BingSearch`
-
-For example, to change to the Brave Search API, you would configure it as follows:
-
-```python
-BingBrowser(
-    searcher_type='BraveSearch',
-    topk=2,
-    api_key=os.environ.get('BRAVE_API_KEY', 'YOUR BRAVE API')
-)
```

Comment:
Why remove？

Reply:
bacause someone alr supported the function that change different type of seach engine in CLI, refer to [This PR](https://github.com/InternLM/MindSearch/pull/141), so this hard code way is not needed anymore, now  just add "----search_engine BraveSearch" when running mindsearch.app will having the same effect。

Reply:
@Harold-lkk may i ask does this pr is able to merge?